### ISSUE TITLE
USWDS - Create USA Icons package that generates a Sprite

### ISSUE THREAD

**Issue by brunerae** [uswds/uswds#4867]
Goal: have USA Icons do the same thing that current USA package does (include default sprite, take config file, generate Sprite). Update USWDS package to pull-in Sprite.

**Comment by amyleadem** [comment]
Goal: Users should be able to access all USWDS icon assets via the usa-icon package.

Expected behavior: 
- The commands `gulp buildSprite`, `gulp buildSpriteStandalone`, and `gulp buildUSWDS` should output the same assets to both `./dist` and `./packages/usa-icon/dist`.
- These assets should match what is currently built in the `./dist` directory in the `develop` branch.

## Reference
### Output to `./dist` (from current `develop` branch)
- `gulp buildSprite`
    
    ![image](https://user-images.githubusercontent.com/93996430/180801195-c5267a00-ab9d-4385-86d9-044a0a660567.png)
- `gulp buildSpriteStandalone`
    
    ![image](https://user-images.githubusercontent.com/93996430/180800705-063634f8-db42-4487-b0fa-4f44c1d435b0.png)
- `gulp`
    
    ![image](https://user-images.githubusercontent.com/93996430/180801687-7f0cd835-cfbc-44ed-81ac-37ea0e26870b.png)

**Comment by mejiaj** [comment]
> Exception: we should exclude hero.png from the package assets. Can we remove all .png files?

@amyleadem this is correct, any icons required for the sprite + legacy icons should be included for now. Anything that's not an icon like `hero.png` and favicons can be excluded.

**Comment by amyleadem** [comment]
New task: Create unit test that the correct directories and files exist
Add at least one basic test
Reference: https://github.com/uswds/uswds/blob/develop/src/test/build.spec.js
